nGEMApp/Db/double_param.template

```diff
@@ -0,0 +1,14 @@
+record(ai, "$(P)$(Q)$(NAME)")
+{
+    field(DESC,"$(DESC)")
+    field(DTYP, "asynFloat64")
+    field(INP, "@asyn($(PORT),0,1)$(NAME)_$(TYPE)")
+	field(SCAN, "I/O Intr")
```

Comment:
It looks like some of the items using this template might want archive/interest setting on them?

Reply:
Currently this is run on a separate laptop in a mini-inst like way so there no mysql or archive engine locally. So though they should be added they would not be visible at present.

Reply:
I would need to refactor the templates as currently it would be an all or nothing on archive